include/libaktualizr-c.h

```diff
@@ -0,0 +1,30 @@
+#ifndef AKTUALIZR_LIBAKTUALIZRC_H
+#define AKTUALIZR_LIBAKTUALIZRC_H
+
+#ifdef __cplusplus
+#include "primary/aktualizr.h"
+
+using Campaign = campaign::Campaign;
+
+extern "C" {
+#else
+typedef struct Aktualizr Aktualizr;
+typedef struct Campaign Campaign;
+#endif
+
+Aktualizr *Aktualizr_create(const char *config_path);
+int Aktualizr_initialize(Aktualizr *a);
+int Aktualizr_uptane_cycle(Aktualizr *a);
+void Aktualizr_destroy(Aktualizr *a);
+
+Campaign *Aktualizr_campaign_check(Aktualizr *a);
+int Aktualizr_campaign_accept(Aktualizr *a, Campaign *c);
```

Comment:
That's a bit annoying that we have to pass an Aktualizr instance to all of these. Would be handy if Campaign contained a pointer. But that might need changes in the C++ object, don't know if we want that.

Reply:
I see no problem in this, tbh. That's pretty much idiomatic C and is equivalent to `Aktualizr->CampaignAccept(Campaign *c). If we do just a single arg here, I would expect our C++ API to also change to `Campaign->Accept()`. I'm not sure if it makes it nicer, though.

Reply:
Ok I think you're right.

Reply:
On second thought, maybe it's not a bad thing to make those calls members of Campaign object. This way, we would prevent passing a campaign_id that belongs to another Aktualizr instance (a hypothetical situation, but still). WDYT, @lbonn @patrickvacek @mike-sul ?

Reply:
If it doesn't require changes to the C++ code, then that seems like the way to go. Even if it does, that seems to me to be a slightly better way to go.